home assignment/7/UI.py

```diff
@@ -0,0 +1,56 @@
+import tools as tl
+
+print('Привет! Я телефонный справочник')
+
+def menu():
+    while True:
+        print()
+        print('1. Сгенерировать справочник')
+        print('2. Поиск по фамилии')
+        print('3. Поиск по номеру телефона')
+        print('4. Добавить новую запись')
+        print('5. Изменить запись')
+        print('6. Удалить запись')
+        print('7. Посмотреть справочник целиком')
+        print('8. Выйти из программы')
```

Comment:
Зачем уж так? Почему нельзя сделать многострочный комментарий и использовать один `print`?
```suggestion
text = '''
1. Сгенерировать справочник
2. Поиск по фамилии
3. Поиск по номеру телефона
4. Добавить новую запись
5. Изменить запись
6. Удалить запись
7. Посмотреть справочник целиком
8. Выйти из программы
                '''
def menu():
    while True:
        print(text)
```